apps/spruce/src/pages/projectAndRepoSettings/shared/tabs/GithubCommitQueueTab/getFormSchema.tsx

```diff
@@ -344,7 +351,26 @@ export const getFormSchema = (
         githubPRTriggerAliases: {
           "ui:addable": false,
           "ui:orderable": false,
-          "ui:placeholder": "No GitHub Trigger Aliases are defined.",
+          "ui:placeholder":
+            "No aliases are scheduled to run for pull requests.",
+          "ui:readonly": true,
+          "ui:removable": false,
+          "ui:descriptionNode": (
+            <GithubTriggerAliasDescription
+              identifier={identifier}
+              isPR
+              isRepo={projectType === ProjectType.Repo}
+            />
+          ),
+          items: {
+            "ui:field": "githubTriggerAliasField",
```

Comment:
I don't understand / know how to bind the aliases for the `githubMQTriggerAliases` section below. It currently just doesn't bind (tested on staging). Any advice on this is greatly appreciated!